osu.Server.Spectator/Hubs/MultiplayerHubContext.cs

```diff
@@ -166,15 +167,15 @@ public async Task StartMatch(ServerMultiplayerRoom room)
             if (room.Queue.CurrentItem.Expired)
                 throw new InvalidStateException("Cannot start an expired playlist item.");
 
-            var readyUsers = room.Users.Where(u => u.State == MultiplayerUserState.Ready).ToArray();
-
             // If no users are ready, skip the current item in the queue.
-            if (readyUsers.Length == 0)
```

Comment:
What's the reasoning behind this?

Reply:
This condition is (at the moment) only relevant when countdown is used (if match is started by host, host will be ready).

The reason for the change is that we want `readyUsers` to contain both `Ready` and `Idle` players because gameplay will start for both. But we only want to consider `Ready` players when deciding whether gameplay will start or not. When everyone is idle and countdown reaches 0, it seems foolish to actually start gameplay. In my opinion at least one person should be ready.